app/models/openscap_result.rb

```diff
@@ -29,6 +29,7 @@ def html
   private
 
   def create_rule_results
+    openscap_rule_results.delete_all
```

Comment:
Please move this line to create_results.
That way if arf parsing suddenly fails we will retain the last successful result